src/apps/mesoscale-explorer/data/mmcif/model.ts

```diff
@@ -186,9 +186,10 @@ const MmcifStructure = PluginStateTransform.BuiltIn({
             } else {
                 structure = Structure.create(units);
             }
-
+            // could also use _struct_ref.pdbx_db_accession to point to uniprot with _struct_ref.db_name == UNP
             const label = entities.data.pdbx_description.value(idx).join(', ') || 'model';
-            return new PSO.Molecule.Structure(structure, { label, description: `${a.description}` });
+            const description = '*Entity id* ' + entities.data.id.value(idx) + ' *src_method* ' + entities.data.src_method.value(idx) + ' *type* ' + entities.data.type.value(idx);
```

Comment:
```suggestion
            const description = `*Entity id* ${entities.data.id.value(idx)} *src_method* ${entities.data.src_method.value(idx)} *type* ${entities.data.type.value(idx)}`;
```

Also perhaps consider adding fallbacks with `?? 'fallback value'` as `${undefined}` renders as `undefined` which is not quite desirable.